pkg/operator/externaletcdsupportcontroller/external_etcd_support_controller.go

```diff
@@ -156,11 +159,12 @@ func (c *ExternalEtcdEnablerController) supportExternalEtcdOnlyPod(
 	podConfigMap.Data["forceRedeploymentReason"] = operatorSpec.ForceRedeploymentReason
 	podConfigMap.Data["version"] = version.Get().String()
 
-	enabled, err := ceohelpers.IsExternalEtcdSupport(operatorSpec)
+	// we can prepare the external etcd support already while TNF setup is still running
+	supportExternalEtcd, err := ceohelpers.IsEtcdRunningInCluster(ctx, c.operatorClient)
```

Comment:
@tjungblu do I need to wait for the infra informer to sync before this or is this handled by the operator client